telemetry/generated/identifiers.ts

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// AUTOGENERATED BY glean_parser v8.1.1. DO NOT EDIT. DO NOT COMMIT.
+
+import StringMetricType from "@mozilla/glean/private/metrics/string";
```

Comment:
Hey! Quick fly-by here! @wtfluckey we discourage committing the generated files as they might change between glean updates and linters might complain. 

We recommend wiring the glean build and linters to your build system and run them whenever you build/deploy